spec/acceptance/enable_fips_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper_fips'
+
+hosts.each do |host|
+  describe 'ensure FIPS in enabled' do
```

Comment:
Why not check that FIPS mode matches BEAKER_fips setting?  Would eliminate the need for spec_helper_fips.rb .  Would also allow tests to run whether or not BEAKER_fips was set.